### ISSUE TITLE
Typo in the `openmed.prod.sample` file

### ISSUE THREAD

**Issue by lucavenir** [faisitalia/OpenMed-PoC#58]
The file `openmed.prod.sample` has a typo:

```
{
    "publicHostname": "<your-host-name>",
    "publicPort": 443,
    "publicIp: "<your-public-ip>",
    "mongodbUrl": "mongodb://<your-mongo-username>:<your-mongo-password>@<your-mongo-host-name>:27017/<your-mongo-database-name>",
    "gmailUsername": "<your-gmail-address>",
    "gmailPassword": "<your-gmail-app-password>",
    "ssslCert": "/etc/letsencrypt/live/<your-host-name>/fullchain.pem",
    "sslKey: "/etc/letsencrypt/live/<your-hostname-name>/privkey.pem"
}
```

I'm correcting it now.

**Comment by lucavenir** [comment]
Done in `dev` branch

**Comment by lucavenir** [comment]
There was another typo in here that's probably causing the conference not to work

**Comment by lucavenir** [comment]
Fixed with PR #60
